translations/nl-be.json

```diff
@@ -675,8 +675,8 @@
   "option-others": "Andere",
   "saving-agendaitem-title-message": "Aanpassen van agendapunten",
   "saving-change-message": "Uw aanpassing wordt opgeslagen.",
-  "saving-retracted-agendaitem-title": "Aanpassen van uitstelling",
-  "saving-retracted-agendaitem-message": "Uw aanpassing wordt opgeslagen.",
+  "saving-decision-result-title": "Aanpassen van resultaat beslissing",
+  "saving-decision-result-message": "Uw aanpassing wordt opgeslagen. Het resultaat van de beslissing wordt aangepast",
```

Comment:
Possibly the people from Secretarie will have other suggestions for these labels, but we'll cross that bridge when we get to it.

Reply:
This popup won't be visible for long.
Could revert these back to be vague messaging.